### ISSUE TITLE
servo.py should be recoded if used for Pi Car-B

### ISSUE THREAD

**Issue by bluesailor45** [adeept/adeept_picar-b#2]
Issues: 
probably NOT usable for Pi Car-B: should be indicated and corrected 
1) pwm Values for wheel motor and camera motor are hardcoded in code - values should be get from file "set.txt"
2) turn_left, turn_right functions  does not work correct
3) function look_left(speed) and anlog functions: speed is actually only increment value

**Comment by WilliamWalkerMercier** [comment]
How do you correct turn_left turn_right ?

**Comment by adeept** [comment]
The WEB control interface requires the Raspberry Pi to run the webSever.py program. webServer.py will call the RPiservo.py program to control the servo.
" turn_left , turn_right " is used for server.py program, but server.py program is not recommended. It is more recommended to use the webServer.py program.

**Comment by WilliamWalkerMercier** [comment]
how do i use turn_left turn_right with the web server program ? do i have to change a call ?

**Comment by adeept** [comment]
Yes, you need to call the function in RPiservo.py to control the servo in the WEB interface.


For example: Line 242 in the webServer.py file is to call the function in RPiservo.py:
scGear. moveAngle(2, 30)
Among them, "2" is the port number of the servo, and "30" is the rotation of the servo to 30 degrees (the default initial angle is 90 degrees)


 Technical Support
 ***@***.***
  



---- Replied Message ----
From  ***@***.***> Date  5/10/2023 01:55 To  ***@***.***> Cc  ***@***.***> ,
 ***@***.***> Subject  Re: [adeept/adeept_picar-b] servo.py should be recoded if used for Pi Car-B (#2)  

how do i use turn_left turn_right with the web server program ? do i have to change a call ?

**Comment by WilliamWalkerMercier** [comment]
The GUIfindline program makes some time to turn the wheel, how can i
correct it ?
